src/shared/common/tile_functions.js

```diff
@@ -64,11 +64,11 @@ export function getTileUrlFromCoordsAndTileserver(
         // https://www.maptiler.com/google-maps-coordinates-tile-bounds-projection/
         const newY = 2 ** zoom - y - 1;
         url = formatXYZoomKey(urlTemplate, x, newY, zoom, apiKey);
-    } else if (tileServerName.includes('{-y}')) {
+    } else if (tileServerName === 'custom' && urlTemplate.includes('{-y}')) {
```

Comment:
I think we could just use `else if (urlTemplate.includes`... here. We don't have to check for the tileServerName attribute.

Reply:
ok, updated!